packages/react-components/react-popover/src/stories/Popover/PopoverAnchorToCustomTarget.stories.tsx

```diff
@@ -1,10 +1,6 @@
 import * as React from 'react';
-import { Button } from '@fluentui/react-button';
-import { makeStyles, shorthands } from '@griffel/react';
-
-import { Popover, PopoverTrigger, PopoverSurface } from '@fluentui/react-popover';
-import { PositioningImperativeRef } from '@fluentui/react-positioning';
-
+import { makeStyles, shorthands, Button, Popover, PopoverTrigger, PopoverSurface } from '@fluentui/react-components';
```

Comment:
_nit:_ Order these alphabetically

```suggestion
import { makeStyles, shorthands, Button, Popover, PopoverSurface, PopoverTrigger } from '@fluentui/react-components';
```